<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu_produto</class>
 <widget class="QWidget" name="menu_produto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>291</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>290</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>770</width>
     <height>290</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>770</width>
     <height>290</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lblcodigo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Código</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbldesc">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Descrição</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblqtdisp">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Qtde. Disponível</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblpreco">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Preço Unitário</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblestoquemin">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>150</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Estoque Mínimo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCodigo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDescricao">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>621</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtqtDisp">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPreco">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>150</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEstoquemin">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>150</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>691</width>
      <height>29</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="btnconsultar">
     <property name="text">
      <string>Consultar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnincluir">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Incluir</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnalterar">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Alterar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnexcluir">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Excluir</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnsair">
     <property name="text">
      <string>Sair</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="txtStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>761</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <action name="actionbotao_incluir">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>botao_incluir</string>
   </property>
   <property name="toolTip">
    <string>botao_incluir</string>
   </property>
  </action>
  <action name="actionbotao_excluir">
   <property name="text">
    <string>botao_excluir</string>
   </property>
  </action>
  <action name="actionbotao_sair">
   <property name="text">
    <string>botao_sair</string>
   </property>
  </action>
  <action name="actionbotao_alterar">
   <property name="text">
    <string>botao_alterar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnincluir</sender>
   <signal>clicked()</signal>
   <receiver>actionbotao_incluir</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnexcluir</sender>
   <signal>clicked()</signal>
   <receiver>actionbotao_excluir</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>524</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnsair</sender>
   <signal>clicked()</signal>
   <receiver>actionbotao_sair</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>663</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnalterar</sender>
   <signal>clicked()</signal>
   <receiver>actionbotao_alterar</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
